### ISSUE TITLE
podman pod create --pod-id-file : broken (empty file contents)

### ISSUE THREAD

**Issue by edsantiago** [containers/libpod#6292]
Broken in master (4eee0d840f0af81560d89864ea64b95c9e70b18d):
```console
$ ./bin/podman pod create --name=abc --pod-id-file=/tmp/pod-id-file
a4c2851b4ebc3de399f2feed564c32d76b216c2f6c2df4f7cb8ffd470f434fa8
$ cat /tmp/pod-id-file
[nothing]
```
Works fine in podman-1.9.1-1.fc32

**Comment by rhatdan** [comment]
@sujil02 PTAL

**Comment by sujil02** [comment]
Fixed with #6313
